.travis.yml

```diff
@@ -31,11 +31,15 @@ jobs:
     - stage: linting
       before_script: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.23.3
       script: ./scripts/verify.sh
+      after_success: echo "Linting check succeeded"
+      after_failure: echo "Linting check failed. Run make lint to check it locally"
 
-    - stage: golden
+    - stage: generate
```

Comment:
it's not clear why the name change. The script being called is still called `golden_test.sh`.

Reply:
It feels like `golden` is very specific to what is going on so I would recommend:

1. Changing the stage name back to `golden`
1. Changing the `test-gen` make target to `test-golden`

Reply:
Golden seems like an arbitrary name, generate sounds a bit more descrptive but still no much enhancement there, so I don't really have a strong opinion here.

Reply:
Hi @djzager, 

The problem is that the name golden does not really illustrate what it does.  So:
Golden: run kubebuilder commands to scaffold the projects in the testdata dir
Golden_Test: check if the testdata dir is with all updated ( in other words, if the golden was executed )

So, wdyt bout we rename thee script to generate and generate_test as well? Have you folks a better name to make its purpose be more intuitive?

Reply:
`generate` sounds like a reasonably informative name.

Reply:
Done.